packages/create-sitecore-jss/src/initializers/nextjs-sxa/index.ts

```diff
@@ -29,6 +33,25 @@ export default class NextjsSxaInitializer implements Initializer {
 
     await transform(templatePath, mergedArgs);
 
+    let addThemeProject: string[] = [];
+
+    // prompt to select which theme projects to include
+    if (args.templates.includes('nextjs-sxa')) {
+      const addThemeAnswer = await inquirer.prompt({
```

Comment:
Should we follow same pattern we use for styleguide (our only other add-on with sub-prompts) and move answers/prompts out to `/common/args/sxa.ts` and `/common/prompts/sxa.ts` (see [here](https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/common/args/styleguide.ts) and [here](https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/common/prompts/styleguide.ts) for ref)?

Reply:
As discussed, add-ons are handled directly in the initializer, so we're good here.